apps/asap-server/scripts/migration.template

```diff
@@ -0,0 +1,8 @@
+/* istanbul ignore file */
```

Comment:
`migration.template` => `migration.template.ts` ? Or will the build start complaining?

Reply:
starts moaning, yeah